src/main/java/org/junitpioneer/jupiter/params/RangeSourceProvider.java

```diff
@@ -55,7 +56,7 @@
 								.stream(annotations.annotationType().getAnnotationsByType(ArgumentsSource.class))
 								.anyMatch(annotation -> getClass().equals(annotation.value())))
 						.collect(Collectors.toList()))
-				.get();
+				.orElse(Collections.emptyList());
```

Comment:
It's good to avoid to just `get()`, but there's a comment a few lines above:

> // since it's a method annotation, the element will always be present

This justifies why the `Optional` can never be empty. I think if it is, some central assumption of the extension no longer holds and this _should_ explode. Probably with an `orElseThrow(IllegalStateException::new)`, though.

Reply:
I like the `orElseThrow(IllegalStateException::new)` thing! 👍